pkg/controller/instance/instance_controller_test.go

```diff
@@ -12,6 +12,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/onsi/gomega"
```

Comment:
I know, you just copied it, but could you also replace `gomega` here with `assert`?